dashboard/src/components/AppView/AppControls/UpgradeButton.tsx

```diff
@@ -3,30 +3,27 @@ import * as React from "react";
 import { ArrowUpCircle } from "react-feather";
 
 interface IUpgradeButtonProps {
-  updateVersion?: string;
+  newVersion?: boolean;
   releaseName: string;
   releaseNamespace: string;
   push: (location: string) => RouterAction;
 }
 
 const UpgradeButton: React.SFC<IUpgradeButtonProps> = props => {
-  const { updateVersion, push, releaseName, releaseNamespace } = props;
+  const { newVersion, push, releaseName, releaseNamespace } = props;
   const onClick = () => push(`/apps/ns/${releaseNamespace}/upgrade/${releaseName}`);
   let upgradeButton = (
     <button className="button" onClick={onClick}>
       Upgrade
     </button>
   );
   // If the app is outdated highlight the upgrade button
-  if (updateVersion) {
+  if (newVersion) {
     upgradeButton = (
-      <div className="tooltip">
-        <button className="button upgrade-button" onClick={onClick}>
-          <span className="upgrade-text">Upgrade</span>
-          <ArrowUpCircle color="white" size={25} fill="#82C341" className="notification" />
-        </button>
-        <span className="tooltiptext tooltip-top">New version ({updateVersion}) found!</span>
```

Comment:
I've removed this tooltip text because that info is already explained in the new info

Reply:
Sounds good to me. Do you think the tooltip feature may be used in another place of this app? We may want to keep it for other use cases. However, I see we're using it as an inline CSS style, so we will need to create a component. Definitively, it can be done in a separate task but I wanted to highlight it :)

Reply:
right, for the moment nothing will use it so we don't need it but it's true that moving that to a component can be helpful

_makes a mental note_